CMakeLists.txt

```diff
@@ -36,8 +36,9 @@ separate_arguments(llvm_libs)
 
 enable_testing( )
 
-set(NREACTIONS "5000" CACHE STRING "Number of reactions to simulate")
-set(NSPECIES "2000" CACHE STRING "Number of species to simulate")
+set(NCELLS "100" CACHE STRING "Number of model grid cells")
+set(NREACTIONS "500" CACHE STRING "Number of reactions to simulate")
+set(NSPECIES "200" CACHE STRING "Number of species to simulate")
```

Comment:
It seems we are setting these three variable both here and another header file. Can we only do it once?

Reply:
Before I added the hard-coded jit test these were defined only once in the source files. I added these as cmake variables so that I could use a [shell script](https://github.com/mattldawson/jit_test/blob/main/plots/test.sh) to configure this project to run with differing numbers of reactions and species.

Reply:
Technically, we could remove the definitions from the source files, but leaving the definitions there let's me find the definition of them in a text editor like VS code. The `ifndef` directive means that cmake's values are used instead of what's defined in the source code, but the source has reasonable values. Ultimately it doesn't really matter if they are removed, other than text editors not having any clue where the variable came from.